Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        for ent in incident_details["Entities"][0]:
</pre><p>Same for loop body -&gt; function calls as above:</p><pre class="ql-syntax" spellcheck="false">for entity in incident_details.entities[0]:
   user = _get_user_name(entity)
   if user:
       _show_user_nb(user...)
</pre><p> </p><p>Also notebooklets have this func - you can call this on either the result or notebooklet</p><pre class="ql-syntax" spellcheck="false">if self.check_valid_result_data("related_alerts"):

# e.g.
if ac_nb.check_valid_result_data("account_activity", False):
  display(ac_nb.account_activity)
</pre><p> </p><p>you could prob even wrap in a func:</p><pre class="ql-syntax" spellcheck="false">def _display_acctnb_prop(prop_name, header):
   if ac_nb.check_valid_result_data(prop_name, False):
      md(header)
      display(getattr(ac_nb, prop_nam)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>